src/common/survival_util.cc

```diff
@@ -46,13 +53,13 @@ double AFTNormal::hess_pdf(double z) {
   double pdf;
   double hess;
   pdf     = this->pdf(z);
-  hess = (std::pow(z,2)-1)*pdf;
+  hess = (z*z-1)*pdf;
   return hess;
 }
 
 double AFTLogistic::pdf(double z) {
   double pdf;
-  pdf = std::exp(z)/(std::pow((1+std::exp(z)),2));
+  pdf = std::exp(z)/((1+std::exp(z))*(1+std::exp(z)));
```

Comment:
instead of three calls to exp(z) I would suggest:
```
e_z = std::exp(z);
sqrt_denominator = 1+e_z;
pdf = e_z/(sqrt_denominator*sqrt_denominator);
```

Reply:
Yes. I am doing the changes.

Reply:
I have done the changes.